services/reader.go

```diff
@@ -21,6 +21,19 @@ func ReadLines(file multipart.File) (Output, error) {
 	}
 
 	for _, sheetName := range f.GetSheetMap() {
+		if len(specifiedSheets) > 0 {
+			var findSheet = false
+			for _, v := range specifiedSheets {
+				if v == sheetName {
+					findSheet = true
```

Comment:
can you break the loop if sheet found instead continue all the iterations?

If your `specifiedSheets` has 34 elements and the sheetname is found on the second iteration then you will continue the next 32 iterations for nothing